### PR TITLE
Adding chapter 7 exercises

### PR DESCRIPTION
**Pull request by pratyp**

### CODE REVIEW

File: chapter07/pxd/ex1.pl [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+while (<>) {
+    
+    # Will match 'freddie'
+    if (/fred/) { 
+    	say "The entered line contains the letters fred "; 
+    }
+    
+    # will match only 'fred' 
+    if ( /(^|\s)fred(\s|$)/ ) {
```
Comment (thatguy59) [inline chapter07/pxd/ex1.pl:17]: Nice, I'm assuming that this is to show that you can do both (It does say to print the line if it mentions fred, so if you did it that way then with this you'd get duplicate lines).

File: chapter07/pxd/ex5.pl [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+use strict;
+use warnings;
+use feature "say";
+
+say "Enter a couple of lines (Ctrl-D to exit):";
+
+while (<>) {
+    
+    # Will match any double letters
+    if (/(\w)\1/) {
```
Comment (thatguy59) [inline chapter07/pxd/ex5.pl:12]: There's also `\S` to match any non-whitespace characters, so you could match 00 etc

File: chapter07/pxd/ex1.pl (hunk 1 above)
Comment (ardavey) [inline chapter07/pxd/ex1.pl:17]: This is an interesting way of handling word boundaries without cheating and including something from the next chapter :)

File: chapter07/pxd/ex5.pl (hunk 2 above)
Comment (pratyp) [inline chapter07/pxd/ex5.pl:12]: Yeah, I got the question wrong thinking it was just to print letters.

File: chapter07/pxd/ex1.pl (hunk 1 above)
Comment (pratyp) [inline chapter07/pxd/ex1.pl:17]: @thatguy59  - Yeah, the intention was to print the line if it matched the letters and bare word both.
@ardavey  - After going the roundabout way of figuring this out, I realized they were talking about it in the next chapter :confused:

**Pull request merged**